tests/drivers/tee/tee/src/main.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2023 Epam Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @addtogroup t_driver_xen
```

Comment:
I think it is a copy-paste from another test

Reply:
You. This should be fixed

Reply:
updated